src/createDelete.js

```diff
@@ -43,6 +48,10 @@ const createTask = () => {
 
       taskText.replaceWith(input);
     });
+
+    checkbox.addEventListener('change', () => {
+      toggleCheckbox(taskText, task, checkbox);
+    });
   });
```

Comment:
- [ ] Great job adding n event listener to the checkbox (`change`). However The task object's values for `completed` is updated upon user actions but as I showed you in the Zoom video, Nothing happened in your local storage and  the completed status is always false even when reloading the page. Please fix this.


![image](https://github.com/shm04/to-do-list/assets/79058364/3b0250b9-8120-446f-a273-c9d55485e322)